resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -80,7 +80,8 @@ def _add_deployment_sub_statuses_and_counters():
         sa.Column(
             'sub_environments_count',
             sa.Integer(),
-            nullable=True
+            nullable=True,
```

Comment:
Should it be nullable? With a default of 0 I think it can be made not nullable?

Reply:
I do not think so it should nullable if its default value is set to 0